### DISCUSSION TITLE
Prevent whitespace wrapping in Textbox?

### DISCUSSION THREAD

**Discussion by Tennyleaz** [fabricjs/fabric.js#8628] [Q&A] [answered]
I'm creating a `Textbox` object, while text reaches right border of Textbox and whitespace is inserted, the text is wrapped.
How could I change this behavior?

In the following video I typed `hello world`.

https://user-images.githubusercontent.com/7089271/215665683-ffadf4d4-27f8-4298-809b-36bb3b58b35a.mp4

**Comment by ShaMan123** [comment, marked as answer]
no repro makes it hard to help you.
There are open PRs that enhance textbox with a lot of stuff. Take a look at the text label.

**Reply by ShaMan123** [reply]
But it seems you simply need to set a width value for the textbox

**Reply by Tennyleaz** [reply]
This is my code in fiddle:
https://jsfiddle.net/cry37w6q/

How could I set Textbox to either
1. Force wrap and break word, not extend Textbox width.
2. Or, let Textbox always extend width, only wrap at newline?

**Reply by ShaMan123** [reply]
There is a pr for break work.
Look at it
